sematic/ui/packages/common/src/typeViz/views/hugging_face_reference.tsx

```diff
@@ -0,0 +1,106 @@
+import { OpenInNew } from "@mui/icons-material";
```

Comment:
Hmm, actually most of the file names in the FE code base use either camelCase or PascalCase. Do you want to keep them consistent?

The choice between camelCase or PascalCase is not very clear. But my principle has been that PascalCase usually applies to React Components which have to have the initial letters capitalized. camelCase presents a strong intention to remain internal to the package. (there are always exceptions)

we also didn't capture this in the `prompt_response.tsx` PR.

Reply:
fixed (went ahead and fixed prompt_response as well)